gunicorn/arbiter.py

```diff
@@ -535,7 +536,8 @@ def spawn_worker(self):
                 worker.tmp.close()
                 self.cfg.worker_exit(self, worker)
             except:
-                pass
+                self.log.error("Exception during worker exit:\n%s",
```

Comment:
Thanks for feedback!

> since this error is harmless

Depends on what people do in their handler :-) Could be pretty critical to them.

However, I'm also good with level `WARNING`: in practice, people should not swallow that level, and if they do -- too bad.

Since `Logger.exception` [logs on ERROR level](https://docs.python.org/2/library/logging.html#logging.Logger.exception), this is not an option. So, I'll simply switch to `log.warning(...)`.